### DISCUSSION TITLE
THANK YOU!

### DISCUSSION THREAD

**Discussion by silidev** [tadashi-aikawa/obsidian-another-quick-switcher#203] [General]
This is a big help! Found it through https://forum.obsidian.md/t/remove-strict-word-order-requirement-in-quick-switcher-and-link-suggestions/10966/16

**Comment by tadashi-aikawa** [comment]
Sorry for the delayed response, @silidev. :)

I'm glad to hear that!

**Reply by scpedicini** [reply]
@tadashi-aikawa Just wanted to give a shout-out as well. This plugin with its fast support for titles/headings/etc. makes the searching effortless. Thanks so much for building it!
